pytest_tests/testsuites/services/http_gate/test_http_system_header.py

```diff
@@ -0,0 +1,413 @@
+import calendar
+import datetime
+import logging
+import os
+import time
+from typing import Optional
+
+import allure
+import pytest
+from container import create_container
+from epoch import get_epoch, tick_epoch
+from file_helper import generate_file
+from http_gate import (
+    attr_into_str_header_curl,
+    get_object_and_verify_hashes,
+    try_to_get_object_and_expect_error,
+    upload_via_http_gate_curl,
+)
+from pytest import FixtureRequest
+from python_keywords.neofs_verbs import get_netmap_netinfo, head_object
+from wellknown_acl import PUBLIC_ACL
+
+from helpers.storage_object_info import StorageObjectInfo
+from steps.cluster_test_base import ClusterTestBase
+
+logger = logging.getLogger("NeoLogger")
+EXPIRATION_TIMESTAMP_HEADER = "__NEOFS__EXPIRATION_TIMESTAMP"
+EXPIRATION_EPOCH_HEADER = "__NEOFS__EXPIRATION_EPOCH"
+EXPIRATION_DURATION_HEADER = "__NEOFS__EXPIRATION_DURATION"
+EXPIRATION_EXPIRATION_RFC = "__NEOFS__EXPIRATION_RFC3339"
+NEOFS_EXPIRATION_EPOCH = "Neofs-Expiration-Epoch"
+NEOFS_EXPIRATION_DURATION = "Neofs-Expiration-Duration"
+NEOFS_EXPIRATION_TIMESTAMP = "Neofs-Expiration-Timestamp"
+NEOFS_EXIPRATION_RFC3339 = "Neofs-Expiration-RFC3339"
+
+
+@pytest.mark.sanity
+@pytest.mark.http_gate
+class Test_http_system_header(ClusterTestBase):
+    PLACEMENT_RULE = "REP 2 IN X CBF 1 SELECT 2 FROM * AS X"
+
+    @pytest.fixture(scope="class", autouse=True)
+    @allure.title("[Class/Autouse]: Prepare wallet and deposit")
+    def prepare_wallet(self, default_wallet):
+        Test_http_system_header.wallet = default_wallet
+
+    @pytest.fixture(scope="class")
+    @allure.title("Create container")
+    def create_container(self):
+        return create_container(
+            wallet=self.wallet,
+            shell=self.shell,
+            endpoint=self.cluster.default_rpc_endpoint,
+            rule=self.PLACEMENT_RULE,
+            basic_acl=PUBLIC_ACL,
+        )
+
+    @pytest.fixture(scope="class")
+    @allure.title("return_epoch_duration in seconds")
+    def return_epoch_duration(self) -> int:
```

Comment:
naming is incorrect.
`return_epoch_duration` -> `epoch_duration`

Reply:
done